### DISCUSSION TITLE
Odd things going on with a clean install of RC2

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1341] [Q&A] [answered]
So I decided to do a clean install of RC2 Platform.  

I created my DB. 
Set up IIS.
Extracted contents of  Vanjaro_Platform_1.0.5_x64_Install.zip into my root directory.
Set folder permissions.
Navigated to the new website to install.
Filled out all the info and the site was installed. 
When I clicked to proceed to website I was greeted with the following page:
![image](https://user-images.githubusercontent.com/6786299/119866092-07579a00-bee2-11eb-8f2d-6e5c68129fbc.png)
It was a total mess. :-(
I clicked  My Account and got this:
![image](https://user-images.githubusercontent.com/6786299/119866268-4259cd80-bee2-11eb-9c0a-28427e844cf9.png)

Same thing no matter what browser I use.
Was able to login to this:
![image](https://user-images.githubusercontent.com/6786299/119866740-c14f0600-bee2-11eb-9418-c961db6380e0.png)

It has my avatar from somewhere and no way for me to change it.

When I click to open the menu bar the system hangs. (I've reported this once but now have different errors I will report that in a bug.)
![image](https://user-images.githubusercontent.com/6786299/119867197-41756b80-bee3-11eb-8c13-4f003f3c9f0f.png)refresh browser and all looks ok...

Again this is a brand new install nothing done other than what I stated above. So Mandeep what of all this should I report as a bug that has NOT been addressed already.

**Comment by scottw12** [comment]
UPDATE: When I log out it reverts back to bad formatting

![image](https://user-images.githubusercontent.com/6786299/119867830-f019ac00-bee3-11eb-91ea-209876f72941.png)
 What have I missed in my install?

**Comment by scottw12** [comment]
I haven't found an answer to this yet.   I have deleted everything related to this site in ISS, SQL and files in the root. Then I have reinstalled. 
Site is installed and when logged in CSS looks great. After logging out and refreshing the page MOST of the CSS does NOT load. 
Check it out  https://upperdeck-woodworks.com  installed from scratch manually. Look at what load in in chrome developer.  uggg Maybe I need some sleep lol

**Reply by mksmanish79** [reply]
Did you install this on a 32-bit server? Looks like some CSS files are not compiled.

Try by  Design > Themes > Basic >Site > Click on Save

**Reply by scottw12** [reply]
No not running on a 32 bit machine.....
![image](https://user-images.githubusercontent.com/6786299/120256365-8f77cf80-c253-11eb-81f5-e88c0cbcd783.png)

Have run all versions so far.  As I recall this happened to me once before on RC1 but cleared itself up eventually.  Not sure what I may have done...

I went to the basic theme and even changed a couple settings to make sure it had to recompile.. And it made no difference.
I have turned on Debugging and in the chrome developer tools I see no bootstrap classes loaded up.  The only css items loaded seems to be skin.css.

This is all I see loading up:

![image](https://user-images.githubusercontent.com/6786299/120256972-e92cc980-c254-11eb-9b06-3cdf9b02c7d0.png)

**Reply by mksmanish79** [reply]
Please check if Theme.css files exist at Website\Portals\0\vThemes\Basic folder.

**Reply by scottw12** [reply]
There is a Theme.scss but not a Theme.css.

**Reply by mksmanish79** [reply]
Try by Design > Themes > Basic >Site > Click on Save icon > check if green success message comes in the right bottom else check site log for any error.

**Reply by scottw12** [reply]
Green success and no error.  https://screencast-o-matic.com/watch/cr1V2fVhTfn

**Reply by mksmanish79** [reply]
Please clear your logs folder and share your Website\Portals\_default\Logs latest files.

**Reply by scottw12** [reply]
[2021.06.03.log](https://github.com/vanjarosoftware/Vanjaro.Platform/files/6594515/2021.06.03.log)

**Reply by mksmanish79** [reply]
No clue in log file. Please check if dart.exe and sass.snapshot exits in your bin folder.

**Comment by scottw12** [comment, marked as answer]
OK. I went in and not only deleted the websites and SQL database. Even deleted the app-pool. Re-installed this time it installed and runs properly. I will do some more digging, but I think it may have been a problem with security of the app-pool. I have not verified that yet as there was never any error logs involved. None of the system, application, security, IIS showed any errors or warnings... Was very odd... Anyway lets close this thread and move on from this point knowing that if it happens again look deep into security issues. I thank you very much for your time.
